main/main.c

```diff
@@ -959,12 +960,26 @@ PHPAPI void php_html_puts(const char *str, size_t size)
 }
 /* }}} */
 
+#define CALL_ERROR_HOOKS(HOOK_LIST) for ( \
```

Comment:
I have unpleasant experience with `va_list` and I guess I would have to `va_copy`/`va_end` once more if I am using a function rather than a macro, but more important than that, gcc has/had issues with `va_list` being passed to an inline function. Additionally, I guess that it wouldn't be inlined at all in the end, as it's just a hint for the compiler.